README.md

```diff
@@ -24,28 +24,39 @@ Using [purescript-argonaut-core](https://github.com/purescript-contrib/purescrip
 
 ```purescript
 someObject =
```

Comment:
This part of the example could be updated to use the combinators to define an instance of `EncodeJson`, as that's a more common way to build these than this explicit `jsonSingletonObject` and so on.